runtime/src/bank.rs

```diff
@@ -4652,6 +4657,19 @@ impl Bank {
                 .is_active(&feature_set::simple_capitalization::id())
     }
 
+    // Get the current processing bank if the processing time hasn't expired
+    pub fn is_bank_still_processing_txs(
```

Comment:
The name and comment is kind of confusing to me. Seems like it should be `should_bank_still_be_processing_txs` and the comment like "See if the wallclock time from bank creation to now has exceeded the slot time".